FogCreek/Glitch-Community
# Intriguing adjustment: inline EntityPageProjects into User & Team pages

## Links
* Remix link: https://intriguing-adjustment.glitch.me
* Manuscript link https://glitch.manuscript.com/f/cases/3328378/

## Changes:
* Remove EntityPageProjects presenter, and inline into User & Team pages. This presenter doesn't really provide any useful abstractions now that we have ProjectsList.

## How To Test:
* https://intriguing-adjustment.glitch.me
* sign in by setting localStorage.cached_user
* verify that you can feature, pin & unpin projects on your own or on `@community`'s profile
* verify that you cannot feature, pin or unpin others' projects on other user and team profiles (though you can still add them to collections)
